doc/classes/UndoRedo.xml

```diff
@@ -154,6 +169,12 @@
 				Redo last action.
 			</description>
 		</method>
+      <signals>
```

Comment:
Indentation is wrong, you can run `godot --doctool .` to make sure that your template is properly formatted (should also fix the extra newline added at the end of the file).